contracts/access/SignatureBouncer.sol

```diff
@@ -36,6 +49,24 @@ contract SignatureBouncer is Ownable, RBAC {
     _;
   }
 
+  /**
+   * @dev requires that a valid signature with a specifed method of a bouncer was provided
+   */
+  modifier onlyValidSignatureAndMethod(bytes _sig)
```

Comment:
I would use `_signature` here too. But for consistency, maybe it's better to leave it and update it in a future branch.

Reply:
let's follow this up with a change to `_signature`, yeah.